src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getDefaultColor() {
+        return Colors.WHITE.toString();
+    }
+
+    public String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);

```

Comment:
lets make Random as class level variable, not to create new instance of Random class every time we call getRandomColor() method